### ISSUE TITLE
Feature request: hot key to fill in the data.

### ISSUE THREAD

**Issue by pkazmierski** [perfectapi/CKP#108]
Would it be possible to add a personally definable hotkey to flll in the data?

**Comment by DukeyToo** [comment]
I don't think that Chrome supports that, no.

**Comment by PaddeK** [comment]
It seems chrome does support it. Have a look [HERE](https://developer.chrome.com/extensions/commands).
You can define up to 4 commands with default (suggested) 
keys and numerous commands without suggested keys.

Thats quite a nice feature.. i think as soon as i have some
free time i will PR it :)

**Comment by DukeyToo** [comment]
CPK already supports a key combination to pop open - <kbd>Ctrl-Shift-Space</kbd> - what is not supported by Chrome is the "personally definable" part of it.

**Comment by solikeumyeah** [comment]
An AutoFill hotkey would be excellent. It's my #1 wish upon first test of CKP. But would it be any better than AutoFilling automatically after page load - no hotkey required?

**Comment by HowellBP** [comment]
Another extension, [chromeIPass](https://chrome.google.com/webstore/detail/chromeipass/ompiailgknfdndiefoaoiligalphfdae), uses the KeePassHTTP plugin with a KeePass local client and has a hotkey for automatically filling data. I don't know if this is a feature of chromeIPass or KeePassHTTP though. I think that's the functionality that would be desired with CKP.

**Comment by PaddeK** [comment]
Yep i know both.. keepasshttp is a plugin for keepass to transmit/exchange data via http.
ChromeIPass is a chrome extension thats using keepasshttp. The hotkey feature is most likely from chromeIPass.. maybe keepass itself.
